### PR TITLE
🖼 refactoring layout

### LINKED ISSUE #1343: Add Base layout component
### Is your feature request related to a problem?

@roiLeo suggested in #1339 many useful layout features.

Basically I copy this layout from a page to page and It would be nice to have a component:

```vue
<BaseLayout>
 <my-current-component-that-i-am-working-on />
 </BaseLayout>
 
```

### Describe the solution you would like

Basically it should be:

```html

<div class="columns is-centered">
  <div class="column is-half">
  <section class="section">
    <slot ></slot>
  </section>
  </div>
 </div>
 ```

### Describe alternatives you have considered

:roc

### Screeenshots

Paste your vision

### PR DESCRIPTION
**Pull request by roiLeo**
### PR type
- [x] Bugfix
- [x] Refactoring

### What's new?
- [x] PR closes #1343 
- [x] PR cloeses #1359 
- changed pages to use default layout
- new layout for centered half column
- new layout full width

### CODE REVIEW

**Comment by yangwao** [comment]
Wondering if it saves some build minutes, even little!

**Comment by roiLeo** [comment]
> Wondering if it saves some build minutes, even little!

Sure it will, less files to build.

Found some layout that didn't have full height (Transfer) and need some minor changes

File: styles/global.scss [hunk 1]
```diff
@@ -120,7 +120,7 @@ body {
         color: $primary !important;
     }
     .navbar-start  {
-        .router-link-active {
+        .nuxt-link-active {
```
[the remark this answers is not in the document]
Comment (roiLeo) [reply styles/global.scss:123]: keep quiet, weirdo

**Comment by yangwao** [comment]
https://user-images.githubusercontent.com/5887929/144438688-93930985-dc7d-473b-a637-a7b331d043b5.mov

**Pull request merged**